win32ss/user/user32/windows/cursoricon.c

```diff
@@ -2651,10 +2926,13 @@ HICON WINAPI CreateIconFromResourceEx(
     if(isAnimated)
         HeapFree(GetProcessHeap(), 0, cursorData.aspcur);
 
+end:
+    HeapFree(GetProcessHeap(), 0, pbBmpIcon);
     return hIcon;
 
     /* Clean up */
 end_error:
+    HeapFree(GetProcessHeap(), 0, pbBmpIcon);
     if(isAnimated)
         HeapFree(GetProcessHeap(), 0, cursorData.aspcur);
     DeleteObject(cursorData.hbmMask);
```

Comment:
```suggestion
    if (cursorData.hbmMask) DeleteObject(cursorData.hbmMask);
```
and for the next ones:
```diff
-    if(cursorData.hbmColor) DeleteObject(cursorData.hbmColor);
+    if (cursorData.hbmColor) DeleteObject(cursorData.hbmColor);
-    if(cursorData.hbmAlpha) DeleteObject(cursorData.hbmAlpha);
+    if (cursorData.hbmAlpha) DeleteObject(cursorData.hbmAlpha);
```

Reply:
Changes made. Thanks.